.github/ISSUE_TEMPLATE/error-report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Error report
+about: Used to report production errors seen in AMP Error Reporting.
```

Comment:
Since we've stopped using the word "production" to refer to our channels, how about something like this?

> Used to report errors seen in AMP's release channels.

Reply:
Done